client/src/javascript/components/modals/feeds-modal/DownloadRulesTab.js

```diff
@@ -112,15 +134,42 @@ class DownloadRulesTab extends React.Component {
     150
   );
 
+  checkMatch(match, exclude, check){
+    let checkMatchTextbox;
+    for (let i = 0; i < this.formRef.formRef.length; i++) {
+      let element = this.formRef.formRef[i];
+      if (element.name === 'check'){
+        checkMatchTextbox = element;
+      }
+    };
+
+    if (!Validator.isNotEmpty(match) || 
+      !Validator.isRegExValid(match) || 
+      !Validator.isRegExValid(exclude)){
+      checkMatchTextbox.style = {};
+      return;
+    }
+
+    const isMatched = (new RegExp(match, 'gi')).test(check);
+    const isExcluded = exclude !== '' && (new RegExp(exclude, 'gi')).test(check);
+    
+    if(isMatched && !isExcluded){
+      checkMatchTextbox.style.background = '#39ce83';
+    } else {
+      checkMatchTextbox.style.background = '#e95779';
+    }
+  }
```

Comment:
We shouldn't be setting styles inline. Also, why do we need to change the styles at all?

Reply:
It was neccesary to change the style there in order to indicate whether the given example matches the supplied patterns. 
The behaviour is shown in the last image of the imgur link. 
If the given example matches the patterns, the background will be #39ce83, otherwise #e95779.

What would be the best way to apply the styling in this case?

Reply:
Ahh okay, I'd prefer to use CSS to style it. You can apply classes conditionally using `classnames`